pkg/apiserver/rest/metrics.go

```diff
@@ -10,43 +10,10 @@ import (
 )
 
 type dualWriterMetrics struct {
-	legacy        *prometheus.HistogramVec
-	storage       *prometheus.HistogramVec
-	outcome       *prometheus.HistogramVec
```

Comment:
the outcome metric tries to compare the legacy <> unified results.  It is useful sometimes, but our other integration testing does the same thign

Reply:
As implemented... I also do not really trust it :) and it does not appear to be in any key dashboard